rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -251,7 +251,7 @@ bool lv2_fs_mount_info_map::vfs_unmount(std::string_view vpath, bool remove_from
 
 std::string lv2_fs_object::get_normalized_path(std::string_view path)
 {
-	std::string normalized_path = std::filesystem::u8path(path).lexically_normal().string();
+	std::string normalized_path = std::filesystem::path(path).lexically_normal().string();
```

Comment:
why is this changed?

Reply:
I fixed a warning:

```
[1/4] Building CXX object rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/Cell/lv2/sys_fs.cpp.obj
C:/src/rpcs3/rpcs3/Emu/Cell/lv2/sys_fs.cpp: In static member function 'static std::string lv2_fs_object::get_normalized_path(std::string_view)':
C:/src/rpcs3/rpcs3/Emu/Cell/lv2/sys_fs.cpp:254:62: warning: 'std::filesystem::__cxx11::path std::filesystem::__cxx11::u8path(const _Source&) [with _Source = std::basic_string_view<char>; _Require = path; _CharT = char]' is deprecated: use 'path((const char8_t*)&*source)' instead [-Wdeprecated-declarations]
  254 |         std::string normalized_path = std::filesystem::u8path(path).lexically_normal().string();
      |                                       ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
In file included from C:/msys64/mingw64/include/c++/13.1.0/filesystem:47,
                 from C:/src/rpcs3/rpcs3/Emu/Cell/lv2/sys_fs.cpp:17:
C:/msys64/mingw64/include/c++/13.1.0/bits/fs_path.h:831:5: note: declared here
  831 |     u8path(const _Source& __source)
      |     ^~~~~~
```

Reply:
Fix is wrong though, you need to provide a string of char8_t

Reply:
Reverted. I started looking into it, but it's so complicated that I'd rather revert than learn about UTF-8 support in C++-20.